src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -99,5 +124,45 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             });
 //#endif
         }
+#if CertificateAuth
+        // TODO: remove this middleware method when the web API authentication NuGet package gets updated and the client certificate gets loaded in the CertificateAuthenticationFilter.
```

Comment:
When the web API authentication NuGet package is released; we can remove this since it's already included in the certificate authentication filter.